.github/workflows/build-test.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
           ./.sonar/scanner/dotnet-sonarscanner begin \
           /k:"Qowaiv_Qowaiv" \
           /o:"qowaiv" \
-          /d:sonar.token="${{ secrets.SONAR_TOKEN }}" \
+          /d:sonar.token="$SONAR_TOKEN" \
```

Comment:
Strange thing is that on a new empty commit, the pipeline ran successfully. After that I did change this syntax, as is [recommended](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions#using-secrets-in-a-workflow).